src/pages/JobCalendar.page

```diff
@@ -155,7 +155,7 @@
                                     strUrl = '{!$Site.CurrentSiteUrl + IF(fPersonalSite, ns+'PersonalSiteJobListing', ns+'VolunteersJobListingFS')}' +
                                         '?Calendar=1&volunteerShiftId=' + strShiftId + '&jobId=' + strJobId +
                                         '&dtMonthFilter=' + strApexDateUTC(dtStart) +
-                                        '{!JSINHTMLENCODE(IF(strParams != null, '&' + strParams, ''))}';
+                                        '{!JSENCODE(IF(strParams != null, '&' + strParams, ''))}';
```

Comment:
@seanpat09 - I added you as a reviewer to the calendar parameter url. Scott got pulled over to case management so I took over this branch and added the fix for easier testing since they both issues need to be tested on the calendar page. Would you mind giving this PR a review? Thanks!